components/core/src/env.rs

```diff
@@ -128,6 +128,19 @@ macro_rules! env_config_duration {
     };
 }
 
+#[macro_export]
+macro_rules! env_config_int {
```

Comment:
It would be nice to add some documentation about what this does, since a quick glance at the implementation doesn't really help all that much.

Reply:
Good call. I'll add docs to the macro definition

Reply:
Done in Does https://github.com/habitat-sh/habitat/pull/6565/commits/83d38218a723c2dc8758817c709dfde6bea7f65c